cloudbrain/rt_server/rt_server.py

```diff
@@ -41,17 +43,19 @@ class RtStreamConnection(SockJSConnection):
 
     def __init__(self, session):
         super(self.__class__, self).__init__(session)
-        self.subscribers = {}
+        self.subscribers = recursivedict()
 
 
-    def send_probe_factory(self, metric_name):
+    def send_probe_factory(self, device_id, device_name, metric):
```

Comment:
Do you mind adding a doc string? See example I sent you. Documenting all methods would be helpful.